lib/reporter.js

```diff
@@ -109,15 +109,15 @@ class JunitReporter extends events.EventEmitter {
                         .time(suite.duration / 1000)
                         .property('specId', specId)
                         .property('suiteName', suite.title)
-                        .property('capabilities', capabilities.sanitizedCapabilities)
+                        // .property('capabilities', capabilities.sanitizedCapabilities)
                         .property('file', spec.files[0].replace(process.cwd(), '.'))
 
                     for (let testKey of Object.keys(suite.tests)) {
                         if (testKey !== 'undefined') { // fix cucumber hooks crashing reporter
                             const test = suite.tests[testKey]
                             const testName = this.prepareName(test.title)
                             const testCase = testSuite.testCase()
-                                .className(`${packageName}.${suiteName}`)
+                                .className(`${suiteName}`)
```

Comment:
why?

Reply:
This should also be an option, not a default.

Reply:
the same.